ploneorg/core/profiles/default/workflows/foundation_member_workflow/definition.xml

```diff
@@ -10,8 +10,34 @@
  <state state_id="approved" title="Approved member">
   <exit-transition transition_id="reject"/>
   <exit-transition transition_id="resign"/>
+  <exit-transition transition_id="suspend"/>
+  <exit-transition transition_id="retire"/>
+  <permission-map name="Access contents information"
+                  acquired="True">
+   <permission-role>Anonymous</permission-role>
+   <permission-role>Manager</permission-role>
+   <permission-role>Owner</permission-role>
+  </permission-map>
+  <permission-map name="Change portal events"
```

Comment:
Lordy, this is the weirdest permission to be associated with non-event types. Like the human tailbone.